docling/datamodel/base_models.py

```diff
@@ -201,6 +202,13 @@ class AssembledUnit(BaseModel):
     headers: List[PageElement] = []
 
 
+class TextImageEnrichmentElement(BaseModel):
+    element: TextItem
```

Comment:
We could generalize this to be `DocItem`, and rename the class `ItemAndImageEnrichmentElement`.